Apps/WebClient/src/ClientApp/src/components/report/immunizationHistory.vue

```diff
@@ -215,6 +212,9 @@ export default class ImmunizationHistoryReportComponent extends Vue {
                             <b-col>Lot Number</b-col>
                         </b-row>
                     </b-col>
+                    <b-col data-testid="immunizationProviderTitle" class="col-2"
+                        >Provider / Clinic</b-col
```

Comment:
@ninosamson on 9978 it askes `Provider/Clinic` (without spaces), which one do you prefer?

Reply:
@leobcgov Let's keep it like that for now.